crud.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy.orm import Session
+
+import models
+from models import DBAuthor, DBBook
+from schemas import AuthorCreate, BookCreate
+
+
+def get_author_list(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author(db: Session, author_id: int):
+    return db.query(DBAuthor).filter(DBAuthor.id == author_id).first()
+
+
+def get_author_by_name(db: Session, name: str):
+    return db.query(DBAuthor).filter(DBAuthor.name == name).first()
+
+
+def create_author(db: Session, author: AuthorCreate):
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_book_list(
+    db: Session,
+    author_id: int | None = None,
```

Comment:
The use of `|` for type hinting is only supported in Python 3.10 and later. If you need to support earlier versions, consider using `Optional[int]` from the `typing` module instead.